### PR TITLE
added cpc pay page

### PR DESCRIPTION
**Pull request by rarcif**

### CODE REVIEW

File: docs/play/cdc-pay-integration.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    ```sh
+    ngrok http 4567
+    ```
+5. Add the forwarding url from the ngrok GUI as your payload endpoint in your merchant dashboard.
+    <img src="./assets/webhook/webhook-step-1.png" />
+6. Run the CDC Pay API from a client (e.g. the webview module in Unity).
+    <img src="./assets/webhook/webhook-step-2.png" />
+
+## CDC Pay documentation
+You can read more about the CDC PAy integration [here](https://pay-docs.crypto.com/?shell#overview-home).
```
Comment (cdc-william) [inline docs/play/cdc-pay-integration.md:194]:
```suggestion
You can read more about the CDC Pay integration [here](https://pay-docs.crypto.com/?shell#overview-home).
```

File: docs/play/cdc-pay-integration.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    git clone https://github.com/cronos-labs/play-cpp-sdk.git
+    ```
+2.  Run the following commands:
+    ```sh
+    cd demo
+    npm install
+    node server.js
+    ```
+    ::: tip
+    Please ensure to repalce the `SIGNATURE_SECRET` from you merchants dasboard after adding the webhook endpoint.
```
Comment (cdc-william) [inline docs/play/cdc-pay-integration.md:182]:
```suggestion
    Please ensure to repalce the `SIGNATURE_SECRET` from you merchants dashboard after adding the webhook endpoint.
```

**Review by cdc-william — COMMENTED** [review]
Some typos found

File: docs/play/cdc-pay-integration.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        {
+            WWWForm form = new WWWForm();
+            form.AddField("amount", "1000");
+            form.AddField("currency", "USD");
+            form.AddField("description", "Product Name");
+            form.AddField("return_url", "http://YOUR_WEBSITE.com/orders/123/complete");
+            form.AddField("cancel_url", "http://YOUR_WEBSITE.com/orders/123/fail");
+
+            using (UnityWebRequest www = UnityWebRequest.Post("https://pay.crypto.com/api/payments", form))
+            www.SetRequestHeader("Authorization", "Bearer SECURITY_KEY");
```
Comment (tomtau) [inline docs/play/cdc-pay-integration.md:54]: I guess good to highlight this should be the "publishable key", not the "secret key", as it's done in the client

File: docs/play/cdc-pay-integration.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+            WWWForm form = new WWWForm();
+            form.AddField("amount", "1000");
+            form.AddField("currency", "USD");
+            form.AddField("description", "Product Name");
+            form.AddField("return_url", "http://YOUR_WEBSITE.com/orders/123/complete");
+            form.AddField("cancel_url", "http://YOUR_WEBSITE.com/orders/123/fail");
+
+            using (UnityWebRequest www = UnityWebRequest.Post("https://pay.crypto.com/api/payments", form))
+            {
+                www.SetRequestHeader("Authorization", "Bearer pk_test_Lp6yX1M1fyx4zmyAyzLNwH1H");
```
Comment (tomtau) [inline docs/play/cdc-pay-integration.md:119]: load the key from an environment or name it "YOUR_PUBLISHABLE_KEY"?

File: docs/play/cdc-pay-integration.md (hunk 3 above)
Comment (rarcif) [reply docs/play/cdc-pay-integration.md]: fixed

File: docs/play/cdc-pay-integration.md (hunk 4 above)
Comment (rarcif) [reply docs/play/cdc-pay-integration.md]: fixed

File: docs/play/cdc-pay-integration.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+### Overview
+The CDC Pay module makes use of REST API calls to trigger certain actions. These action will create events that are retrievable via webhooks. The following shell command illsutrates how to initiate the CDC Pay payment.
+
+```sh
+POST https://pay.crypto.com/api/payments
+```
+```sh
+curl https://pay.crypto.com/api/payments \
+  -u SECRET_KEY: \
```
Comment (kether-c) [inline docs/play/cdc-pay-integration.md:18]: Suggest to have `{{PUBLISHABLE_KEY}}` instead of `SECRET_KEY:` here, seems more clear.

Comment (rarcif) [reply docs/play/cdc-pay-integration.md:18]: I took this info from the main CDC Pay documentation. Does this work with the Publishable key. If yes I can change it and perhaps suggest this change on the CDC PAY page as well

Screenshot 2022-07-15 at 11 37 23

File: docs/play/cdc-pay-integration.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+```
+
+::: tip
+This API can be called using either your Secret Key or Publishable Key. These keys are generated in the merchant center [dashboard](https://merchant.crypto.com/users/sign_in/).
+:::
+
+## Integrating with Unity
+To use the CDC Pay module in Unity we can simple use the [UnityWebRequest](https://docs.unity3d.com/ScriptReference/Networking.UnityWebRequest.Post.html) module to simulate a post request. We recommend to use a webhook server to retrieve the payment event responses.
+
+- Create payment example
```
[the remark this answers is not in the document]
Comment (rarcif) [reply docs/play/cdc-pay-integration.md]: done

File: docs/play/cdc-pay-integration.md (hunk 5 above)
Comment (tomtau) [reply docs/play/cdc-pay-integration.md:18]: https://pay-docs.crypto.com/#api-reference-resources-payments-create-a-payment
>  This API can be called using either your Secret Key or Publishable Key.

Comment (kether-c) [reply docs/play/cdc-pay-integration.md:18]: Yes I tried It does work with both `KEY`, may be changed to sth like `{{PUBLISHABLE_KEY or SECRET_KEY}}`

Comment (tomtau) [reply docs/play/cdc-pay-integration.md:18]: given this is most likely the untrusted client side, it should be the publishable key

Comment (rarcif) [reply docs/play/cdc-pay-integration.md]: ok than this is updated

**Review by kether-c — APPROVED** [review]
Looks good to me

**Comment by zkaizhi** [comment]
After discussion with @rarcif , we will merge into gitbook branch instead

**Pull request closed without merging**